shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/Delegs.hs

```diff
@@ -157,6 +176,18 @@ delegsTransition = do
       let ptr = Ptr slot txIx (fromIntegral $ length gamma)
       trans @(DELPL crypto) $
         TRC (DelplEnv slot ptr pp acnt, dpstate', c)
+  where
+    -- @wdrls_@ is small and @rewards@ big, better to transform the former
+    -- than the latter into the right shape so we can call 'Map.isSubmapOf'.
+    isSubmapOf :: Network -> Map (RewardAcnt crypto) Coin -> RewardAccounts crypto -> Bool
+    isSubmapOf network wdrls_ rewards = wdrls_' `Map.isSubmapOf` rewards
+      where
+        wdrls_' =
+          Map.fromList
+            [ (cred, coin)
+              | (RewardAcnt network' cred, coin) <- Map.toList wdrls_,
+                network' == network
```

Comment:
Can the network in `wdrls_` actually differ from `liftSTS $ asks networkId`?

Reply:
I don't believe so. The network stuff is just in here because it's needed to serialise as a reward account. So I think it's fine to completely ignore it with this change.